wd.sh

```diff
@@ -361,7 +360,10 @@ zparseopts -D -E \
     c:=wd_alt_config -config:=wd_alt_config \
     q=wd_quiet_mode -quiet=wd_quiet_mode \
     v=wd_print_version -version=wd_print_version \
-    d=wd_debug_mode -debug=wd_debug_mode
+    d=wd_debug_mode -debug=wd_debug_mode \
+    f=wd_force_mode -force=wd_force_mode
+
+local WD_FORCE_MODE=$([[ ! -z $wd_force_mode ]] && echo true || echo false)
```

Comment:
Not too sure how inkeeping this is with style, feedback appreciated

Reply:
When considering this, I was about to write that it could be done in the same way as `wd_debug_mode`, where a local variable with the same name is set with a default value, like so:

```zsh
local WD_CONFIG=${WD_CONFIG:-$HOME/.warprc}
local WD_QUIET=0
local WD_EXIT_CODE=0
local WD_DEBUG=0                                            # <------ default

...

zparseopts -D -E \
    c:=wd_alt_config -config:=wd_alt_config \
    q=wd_quiet_mode -quiet=wd_quiet_mode \
    v=wd_print_version -version=wd_print_version \
    d=wd_debug_mode -debug=wd_debug_mode                    # <------ parsed here
```
But, then I couldn't figure out if the `WD_DEBUG` variable was actually used for anything.. Perhaps it's redundant?

If so, I think the initialization of `WD_FORCE_MODE` is also redundant, and `wd_force_mode` can be used directly. Might have to be modified so the add function so it expects a zero/non-zero value intead of `true`/`false`.

Would you be able to test this?

Also, remember to unset it in the bottom, together with the rest so they don't carry out in the callers scope.